internal/graph/graph.go

```diff
@@ -144,6 +145,8 @@ type NodeMap map[NodeInfo]*Node
 // NodeSet maps is a collection of node info structs.
 type NodeSet map[NodeInfo]bool
 
+type nodePtrSet map[*Node]bool
```

Comment:
Add a comment explaining where this type is useful: To hold sets of nodes for graphs created with the call_tree option, which may have duplicate nodeInfo.